### PR TITLE
Add PSC+UG lists

### PR DESCRIPTION
**Pull request by rbovard**
Fix https://github.com/geomapfish/geomapfish_website/issues/28#issuecomment-325677083

Please review.

If it's ok (content and style), I'll translate it in english but I'll need some help for the german version.

### CODE REVIEW

File: content/pages/1_community_fr.md [hunk 1]
```diff
@@ -37,6 +37,46 @@ La communauté, composée d’un UserGroup qui représente l’organe délibéra
...
+* Bâle-Campagne, Bâle-Ville, Neuchâtel (cantons)
+* Cartoriviera, Nyon (villes)
+* GeoCommunes (privé)
+* Camptocamp (fournisseur)
+
+### UserGroup
+
+Le UserGroup est composé de représentants des entités suivantes :
+
+* ALPGIS SA
```
Comment (kalbermattenm) [inline content/pages/1_community_fr.md]: Not participating anymore  (only for information)

File: content/pages/1_community_fr.md [hunk 2]
```diff
@@ -37,6 +37,46 @@ La communauté, composée d’un UserGroup qui représente l’organe délibéra
...
+* Canton de Bâle-Campagne
+* Canton de Bâle-Ville
+* Canton de Neuchâtel
+* Canton de Schwyz
+* Canton de Thurgovie
```
Comment (kalbermattenm) [inline content/pages/1_community_fr.md]: Not participating (only for information)

File: content/pages/1_community_fr.md [hunk 3]
```diff
@@ -37,6 +37,46 @@ La communauté, composée d’un UserGroup qui représente l’organe délibéra
...
 * Partager les expériences de chacun sur la solution GMF.
 
+### PSC
+
+Le PSC est composé des entités suivantes :
+
+* Bâle-Campagne, Bâle-Ville, Neuchâtel (cantons)
+* Cartoriviera, Nyon (villes)
+* GeoCommunes (privé)
+* Camptocamp (fournisseur)
```
Comment (kalbermattenm) [inline content/pages/1_community_fr.md]:
```suggestion
* Camptocamp SA (fournisseur)
```

**Review by kalbermattenm — CHANGES_REQUESTED** [review]

**Comment by kalbermattenm** [comment]
Great ! Thanks a lot !

I'll post the german content in a few minutes.

**Comment by kalbermattenm** [comment]
Here is the German part:

```
### PSC

Der PSC besteht aus den folgenden Entitäten:

* Basel-Landschaft, Basel-Stadt, Neuenburg (Kantone)
* Cartoriviera, Nyon (Städte)
* GeoCommunes (privat Unternehmen)
* Camptocamp (Anbieter)

### Benutzergruppe

Die Benutzergruppe besteht aus Vertreter von folgenden Entitäten:

* Kanton Basel-Landschaft
* Kanton Basel-Stadt
* Kanton Neuenburg
* Kanton Thurgau
* Kanton Waadt
* Kanton Graubünden
* Kanton Jura
* Kanton Tessin
* CartoJuraLéman
* Gemeinde Nendaz
* Ecole polytechnique fédérale de Lausanne
* Gemetris SA
* HKD Géomatique SA
* Ingéo SA
* Jaquier Pointet SA
* RUDAZ + PARTNER SA
* swisstopo
* Stadt Lausanne
* Stadt Morges
* Stadt Nyon
* Stadt Prilly
* Stadt Pully
* Stadt Vevey
* Stadt Yverdon-les-Bains
```

**Comment by rbovard** [comment]
Thanks for your feedback and the german translation.

I'll add the english one on Monday (and also update pages metadata).

**Comment by kalbermattenm** [comment]
Thanks for your work !

**Comment by rbovard** [comment]
@kalbermattenm Please review.

**Review by kalbermattenm — APPROVED** [review]
Looks good to me.

Feel free to publish. Maybe it would be good to send a global email to the user group about these changes.

Thanks a lot !!

**Pull request merged**